API.md

```diff
@@ -1166,6 +1169,50 @@ module('awesome-sauce', function(hooks) {
 
 Returns **([Array][70]\<Warning> | [Promise][66]<[Array][70]\<Warning>>)** An array of warnings information
 
+## registerHook
```

Comment:
It looks like you hand-authored these additions. Unfortunately, that's going to get stomped as soon as anything is published, because [the API docs are generated from source][1]! You'll need to move these details into the TSDoc comments. 😅 

[1]: https://github.com/emberjs/ember-test-helpers/blob/a037f3080a80fd6c645a6b677c35a6cd9065d2d5/package.json#L32

Reply:
Doh! Thanks for point this out.